src/integrant/repl.clj

```diff
@@ -30,9 +30,11 @@
             (throw (wrap-ex ex halt-ex)))))
       (throw ex))))
 
-(defn- init-system [config]
+(defn- init-system [config & [keys]]
```

Comment:
As this is a private function, I don't think `keys` needs to be an optional argument.